op-bindings/predeploys/addresses.go

```diff
@@ -25,8 +25,15 @@ const (
 	L1FeeVault                    = "0x420000000000000000000000000000000000001a"
 	SchemaRegistry                = "0x4200000000000000000000000000000000000020"
 	EAS                           = "0x4200000000000000000000000000000000000021"
-	CeloRegistry                  = "0x000000000000000000000000000000000000ce10"
-	GoldToken                     = "0x471ece3750da237f93b8e339c536989b8978a438"
+
+	CeloRegistry                      = "0x000000000000000000000000000000000000ce10"
+	GoldToken                         = "0x471ece3750da237f93b8e339c536989b8978a438"
+	FeeHandler                        = "0xcd437749e43a154c07f3553504c68fbfd56b8778"
+	FeeCurrencyWhitelist              = "0xbb024e9cdcb2f9e34d893630d19611b8a5381b3c"
+	MentoFeeHandlerSeller             = "0x4efa274b7e33476c961065000d58ee09f7921a74"
+	UniswapFeeHandlerSeller           = "0xd3aee28548dbb65df03981f0dc0713bfcbd10a97"
+	SortedOracles                     = "0xefb84935239dacdecf7c5ba76d8de40b077b7b33"
+	AddressSortedLinkedListWithMedian = "ED477A99035d0c1e11369F1D7A4e587893cc002B"
```

Comment:
With the exception of the Registry and GoldToken, I don't think there is a need to preserve the current addresses, as those contracts usually (always?) used internally. But keeping the addresses also won't hurt (except for making them less easy to recognize compared to addresses like `0x4200000000000000000000000000000000000021`).

Reply:
Personally I would prefer to keep it like this at least for now since it is easier to troubleshoot any problems with devnet (it can be compared to working mainnet). Since we should be registry to get smart contract addresses, it shouldn't be any problem to change it in later stages.